mmcomplex.raw.html

```diff
@@ -726,19 +725,18 @@
 -->
 
 However, he ran into some technical complications, such as the fact that some
-numbers have two equivalent decimal representations (e.g.  <A
-HREF="0.999....html">0.999...  = 1.000...</A>), that would have made a
+numbers have two equivalent decimal representations
+(e.g. ~ 0.999... = 1.000...), that would have made a
```

Comment:
Here, a reference is within an equation now. Is this a good idea? Maybe "(e.g. 0.999... = 1.000..., see ~ 0.999...)" would be clearer,

Reply:
I wasn't sure about that one either. The page when I started was using a somewhat similar, dubious, way of formatting the link. Anyway, I've adopted this suggestion.